cfdb/log.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import os
+from datetime import datetime
+from pathlib import Path
+
+from rich.logging import RichHandler
+from rich.progress import (
+    BarColumn,
+    MofNCompleteColumn,
+    Progress,
+    TaskProgressColumn,
+    TextColumn,
+    TimeRemainingColumn,
+)
+
+
+def initialize_logging(logging_dir: Path = None):
+    # use package name to affect all descendants
+    logger = logging.getLogger('cfdb')  
+    logger.setLevel(logging.DEBUG)
+
+    if logger.handlers:
```

Comment:
return early if our logger already has handlers. That means someone else configured stuff and we shouldn't mess.